### DISCUSSION TITLE
GET Team logo

### DISCUSSION THREAD

**Discussion by Iwhite67** [PhlexPlexico/G5API#203] [Help]
Hello, this is on server side, on get5 side or client side ?

```Downloading resource/flash/econ/tournaments/teams/2PCKQ.png.
Error downloading resource/flash/econ/tournaments/teams/2PCKQ.png```

**Comment by PhlexPlexico** [comment]
You need to have a FastDL server setup on your server for this, so it's server config side. You can either use a plugin (not recommended) or just setup a quick FastDL/Http server to serve those files as well.

**Comment by Iwhite67** [comment]
Thanks I check

I have put the plugin team logo


_________________________

Alexis Zaun
Technicien SAV Nord Est

FOM Industrie FRANCE

***@***.******@***.***>
(+33) 06 40 62 82 45
________________________________
